### PR TITLE
feat: add mentors stats

### PR DESCRIPTION
**Pull request by ManojBahuguna**
ref #51 

https://github.com/MentorsWithoutBorders/mwb-partners-admin/assets/19573007/d229114e-c897-40ff-9968-69fcb94352fb

### CODE REVIEW

File: src/components/DashboardItem/DashboardItem.styled.ts [hunk 1]
```diff
@@ -29,13 +29,18 @@ export const DashboardItemValue = styled(Typography)(({ theme }) => ({
...
-  justifyContent: 'space-between',
-  alignItems: 'center'
-}))
+export const DashboardItemsWrapper = styled(Box)<{ $isLoading?: boolean }>(
+  ({ $isLoading }) => ({
+    display: 'flex',
+    gap: '20px',
+    flexWrap: 'wrap',
+    padding: '20px',
+    backgroundColor: '#E0E8F8',
```
Comment (edmondpr) [inline src/components/DashboardItem/DashboardItem.styled.ts:38]: Let's add this color code in styles/theme.ts and import it from there.

**Review by edmondpr — DISMISSED** [review]

**Comment by Mohmed-Refaay** [comment]
Great work!!
btw the search inputs are already implemented in this PR https://github.com/MentorsWithoutBorders/mwb-partners-admin/pull/35.

**Comment by ManojBahuguna** [comment]
> Great work!! btw the search inputs are already implemented in this PR #35.

Hi @Mohmed-Refaay, actually I only integrated the api for search. I used the already [implemented](https://github.com/MentorsWithoutBorders/mwb-partners-admin/blob/main/src/pages/app/mentors.tsx#L52) search-input component in [master branch](https://github.com/MentorsWithoutBorders/mwb-partners-admin/blob/main/src/components/Input/InputWithCheckboxes/InputWithCheckboxes.tsx).

File: src/components/DashboardItem/DashboardItem.styled.ts (hunk 1 above)
Comment (ManojBahuguna) [reply src/components/DashboardItem/DashboardItem.styled.ts]: updated..

**Review by Mohmed-Refaay — DISMISSED** [review]

**Review by jfmdev — APPROVED** [review]

**Review by edmondpr — APPROVED** [review]

**Pull request merged**